src/components/stylesheets/DeleteProperty.css

```diff
@@ -0,0 +1,43 @@
+.delete-main-container {
+  padding: 2%;
+}

```

Comment:
![image](https://github.com/PrangonGhose/Earthbnb-frontend/assets/114300121/beb89ee4-ec6d-4943-9ea0-452fae2ac199)

It will be suitable for the design that the page's content is centered, for that you could try adding `width: 100%` to the delete-main-container.

The page content is overflowing so you could try using the `overflow: auto` property on delete-main-container to handle the content, making it scrollable.